### ISSUE TITLE
Root Compare

### ISSUE THREAD

**Issue by rawatrob** [nolimits4web/Template7#20]
how to set this condition with  {{#js_compare "@root.title == about"}}

**Comment by evolutionjay** [comment]
```
{{#each data}}
....
  {{#each children}}
    <span>{{@root.id}}</span>
  {{/each}}
{{/each}}
```
@root.id  can't get anything

**Comment by J05HI** [comment]
@rawatrob @nolimits4web 
I need this too. It's really annoying if you can't use that.
Currently my work around is to set a global var in js before compiling and use it than.
`
var templateVars = {};
templateVars.moduleName = moduleName;
`

**Comment by nolimits4web** [comment]
Fixed in 1.2.5
